src/cljc/rems/common/util.cljc

```diff
@@ -7,6 +7,19 @@
 ;; regex from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation
 (def +email-regex+ #"[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*")
 
+;; regex from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation
+(def +phone-number-regex+ #"^\+[1-9][0-9\s]{4,26}$")
```

Comment:
It's an ok regex but would it be better to allow 5-27 numerals and any number of spaces? Now e.g. `+1234567890123456789012345` is valid but `+123 456 789 012 345 678 901 2345` isn't.

Reply:
true, but I did not want to make it too complex

Reply:
i think it can be like this for now, we can alter it later if we see fit

Reply:
fine by me